### ISSUE TITLE
Assign a topic to an item from a personal viewpoint

### ISSUE THREAD

**Issue by benel** [Hypertopic/Porphyry#50]
#### Which user action should be enabled (or restricted)? For who?

Prerequisite: being logged in.

#### Phase 1
- [ ] Scenarios (Gherkin)
- [ ] Mockups
- [ ] Implementation strategy

#### Phase 2
- [ ] Acceptance tests (Capybara)
- [ ] Implementation

### Phase 3
- [ ] Screencast

**Comment by SorelMboua** [comment]
For the "Greek Vases" project, i think this feature could be really important for the users as it allows them to locate the items(photography, vase) easily using their own viewpoint.

**Comment by UndersCoco** [comment]
It would be interesting for our project "Studying abroad", because for examples, the competences that we assign to a course, can depend on who add it, the university or a student.

**Comment by benel** [comment]
@SorelMboua It is probably important... but the example puzzles me since the location is probably an attribute rather than a topic.

**Comment by UndersCoco** [comment]
Yes, you're right, we just moved the University where is located the course to the attributes.
